app/src/views/GrantRegistryGrantDetail.vue

```diff
@@ -102,6 +99,10 @@ function useGrantDetail() {
     if (!grants.value || Array.isArray(id)) return null; // array type unsupported
     return grants.value[Number(id)];
   });
+  const grantMetadata = computed(() => {
+    if (!grant.value) return null;
+    return metadata.value[grant.value.metaPtr];
+  });
```

Comment:
Seems like a good fit for a nice one-liner! 
```suggestion
  const grantMetadata = computed(() => grant.value ? metadata.value[grant.value.metaPtr] : null);
```

Reply:
good call!